### DISCUSSION TITLE
No commits?

### DISCUSSION THREAD

**Discussion by Unloaded123456** [Abstrat-Technologies/rustedanvil#26] [General]
There are no commits since two days ago. Did something go wrong when fixing bugs?

Edit: I may help if I know how to fix the problem.

**Comment by GloryBax** [comment]
Mate we've been busy. We have a life outside of working on this project.

**Reply by Unloaded123456** [reply]
Even this project would be worked on all day, it looks like it would finish next week or even after because almost all scripts of Forge 1.6.4 are broken.

Edit: There are so many errors / deficiencies to fix.

**Reply by arek-kwapis** [reply]
> Even this project would be worked on all day, it looks like it would finish next week or even after because almost all scripts of Forge 1.6.4 are broken.
> 
> Edit: There are so many errors / deficiencies to fix.

Yes, it will take WAY longer than a week. It could take a month. It could take a year. It could take more than that. We don't know. All we do know is that we're working on it.

Even developers need time off...

**Reply by Unloaded123456** [reply]
> > Even this project would be worked on all day, it looks like it would finish next week or even after because almost all scripts of Forge 1.6.4 are broken.
> > Edit: There are so many errors / deficiencies to fix.
> 
> Yes, it will take WAY longer than a week. It could take a month. It could take a year. It could take more than that. We don't know. All we do know is that we're working on it.
> 
> Even developers need time off...

Expecting it to finish tomorrow would be false hope...

**Reply by GloryBax** [reply]
> Expecting it to finish tomorrow would be false hope...

Yes. Yes it would be insane false hope, you are correct. We've been away from home and therefore away from workstations for two days straight. We're allowed a break and shouldn't be expected to constantly work on things. This is a passion project, not a job that we're getting paid to do.

**Reply by Unloaded123456** [reply]
> > Expecting it to finish tomorrow would be false hope...
> 
> Yes. Yes it would be insane false hope, you are correct. We've been away from home and therefore away from workstations for two days straight. We're allowed a break and shouldn't be expected to constantly work on things. This is a passion project, not a job that we're getting paid to do.

Helping the developers for finding issues would slightly speed up the project but not like x10, even not x2, something like x1.2.
